backend/frontend/src/screens/LoginScreen.js

```diff
@@ -98,8 +98,10 @@ function LoginScreen() {
 
               <label>Password</label>
               <TextField
-                error = {error && errorMessage.length > 0}
-                helperText={ errorMessage.length > 0 && error && errorMessage }
+                error={LoginError && errorMessage.length > 0}
+                helperText={
+                  errorMessage.length > 0 && LoginError && errorMessage
+                }
```

Comment:
Is this helpText is a boolean value or a string value? will this give string result or boolean result? `errorMessage.length > 0 && LoginError && errorMessage`

Reply:
LoginError is boolean value from redux state, if it is true then a error string value is set to errorMessage then it is given to helpertext. helpertext does not display if the errorMessage is empty.